test/test_playbooks/smart_proxy.yml

```diff
@@ -0,0 +1,44 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+  - include: tasks/smart_proxy_puppet_host.yml
+    vars:
+      puppet_host_state: present
+  - include: tasks/smart_proxy.yml
+    vars:
+      smart_proxy_state: absent
+- hosts: tests
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  environment:
+    XDG_CACHE_HOME: "{{ lookup('env', 'FAM_TEST_APYPIE_CACHE_DIR') }}"
+  tasks:
+  - include: tasks/smart_proxy.yml
+    vars:
+      smart_proxy_state: present
+      smart_proxy_download_policy: background
+      expected_change: true
+  - include: tasks/smart_proxy.yml
+    vars:
+      smart_proxy_state: present
+      expected_change: false
```

Comment:
should this expect a change, as you set the download policy to background above, but the default is on_demand?